CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-## 4.1.1 [current]
+## 4.2.0 [current]
+- Add support for [set streaming](https://docs.fauna.com/fauna/current/drivers/streaming).
```

Comment:
Can you add a link to the PR?
`Add support for set streaming [###](link to pr)`

Reply:
this will be customer facing - figured the docs would be more valuable.

Reply:
The documentation link is a good idea, but I agree with Grey that the PR/commit providing the feature should be included.

Reply:
👍